### ISSUE TITLE
Error with RMarkdown: 'is_latex_output' is not an exported object from 'namespace:knitr'

### ISSUE THREAD

**Issue by zxs107020** [rstudio/rmarkdown#1272]
I am getting an error when trying to compile a document using `knitr`. This error occurs when compiling html and/or pdf (havent tried anything else). 

Here is the system information:
```
> sessionInfo()
R version 3.4.1 (2017-06-30)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.3
```

The actual error is:
```
output file: time_tests.knit.md

Error: 'is_latex_output' is not an exported object from 'namespace:knitr'
Execution halted
```
I have searched online but cant seem to find anyone else running into the same issue. Please help.

**Comment by mikey-harper** [comment]
You will need to provide a minimal example to be able to help, and other details would be useful. If you are compiling in RStudio, the console window will show you the progress of the compiling. If it is breaking, this will show which chunk it is being broken in.

As a minimum, you should make sure everything is up to date.  You are using an older version of R: https://stackoverflow.com/questions/13656699/update-r-using-rstudio

Also make sure knitr is the latest version: `install.packages("knitr")`

This question seems more suited for StackOverflow: I would post these extra details there and then copy the link here. If this does turn out to be a bug we can then post more information here later.

**Comment by yihui** [comment]
This has been fixed a couple of days ago via #1271. Thanks!

[When in doubt, please try to upgrade.](https://yihui.name/en/2017/05/when-in-doubt-upgrade/)
